recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
 
 build:
   number: 0
-  skip: True  # [win or osx]
+  skip: True  # [win]
```

Comment:
So any change like this needs to be coupled with a [re-rendering]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ) or it won't work.